sematic/db/migrations/20230621123301_add_organizations_associations.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+
+ALTER TABLE resolutions ADD COLUMN organization_id REFERENCES organizations(id);
+ALTER TABLE runs ADD COLUMN organization_id REFERENCES organizations(id);
+ALTER TABLE metric_labels ADD COLUMN organization_id REFERENCES organizations(id);
+ALTER TABLE artifacts ADD COLUMN organization_id REFERENCES organizations(id);
```

Comment:
This migration is a placeholder. The resulting table is not correct from the SQLAlchemy p.o.v., as explained in #903.

This PR is blocked by #903. Once that PR or its replacement is updated to delete and recreate the tables to leave them in a correct state, the same mechanism will be reused in this PR.

Reply:
This PR has been rebased on top of #903, and the migration was rewritten.